backend/router.js

```diff
@@ -2,16 +2,142 @@ const express = require('express')
+                    between two people. The user is about give their response to a question that you have 
+                    asked. Please provide a relevant reaction to their answer and brief feedback. Remember to keep this concise. Here is the question that was asked: ${context}. Here is the user's response 
+                    to the question: ${message} Lastly, ask a follow-up question to the user based on their response. Make the question relevant to them yet still broad enough to allow them to answer on their own terms. 
+                    Format for the follow up question should be exactly as follows: "As a follow-up question, (the followup question)"`
+                        : !lastQuestion && !doFollowUp ?
+                            `You are a interviewer conducting a behavioral interview. Make sure to talk in the first person as if this was a normal conversation between two people; do not include quotes around your response.  
+                    The user is about give their response to a question that you have 
+                    asked. Please provide a relevant reaction to their answer and say what you liked about their response. Remember to keep this concise. Here is the question that was asked: ${context}. Here is the user's response 
+                    to the question: ${message}. Do not ask any type of question within your response. Say something to end this thought and mention going on to the next question. `
+                            :
+                            `
+                    You are a interviewer conducting a behavioral interview. Make sure to talk in the first person as if this was a normal conversation between two people; do not include quotes around your response.  
+                    The user is about give their response to a question that you have 
+                    asked. Please provide a relevant reaction to their answer and say what you liked about their response. Remember to keep this concise. Here is the question that was asked: ${context}. Here is the user's response 
+                    to the question: ${message}. Do not ask any type of question within your response. Say something to end this thought.
+                    Lastly, since this was the last question, you are done conducting the interview.
+                    You must always express your thanks for getting to interview the person and mention that there will be post interview feedback via our post interview review on our platform InterviewMe. 
+                    `
+                }
+            ],
+            model: "llama3-70b-8192",
+        });
+
+
+        const botResponse = response.choices[0]?.message?.content || "I didn't understand that.";
+
+
+        // Use regex (regular expression) to extract follow-up question
+        const followUpQuestionMatch = botResponse.match(/follow-up question, (.*)$/);
```

Comment:
There is no way to retrieve 2 separate responses within the same question or separate questions? Is this simpler than other alternatives?

Reply:
Maybe we can just use the whole response in the frontend and pass the `doFollowUp` boolean in the response if we need it for additional logic there.